scripts/player.gd

```diff
@@ -1,7 +1,7 @@
 extends KinematicBody2D
 var velocity:Vector2 = Vector2(0,0)
 var up:Vector2 = Vector2(0,-1)
-
+var Health : float = 100.0
```

Comment:
can you have this as health instead of Health

Reply:
Alright, Tree will also will have to make his health variable lowercase

Reply:
Yup this is correct :)